packages/object/src/hashgraph/index.ts

```diff
@@ -316,6 +315,10 @@ export class HashGraph {
 		return true;
 	}
 
+	findNextUnusuallyRelated(hash: Hash, start: number): number | undefined {
```

Comment:
```suggestion
	findNextCausallyUnrelated(hash: Hash, start: number): number | undefined {
```
it was already changed, but reverted?

Reply:
i changed the function name